build.py

```diff
@@ -176,5 +178,10 @@ def build_font_instance(generator, instance_descriptor, *steps):
                     step_merge_nf,
                 )
 
+        print("Autohinting OTFs")
+
+        for file in Path("build").glob("*.otf"):
```

Comment:
This will rehint everything on every instance.

Reply:
Yeah. Probably not the most efficient way to do it. But this way we don't need to be reliant on the uploader running psautohint before sending it up?

Reply:
I think you just need to dedent that block so that it is run after all instances have been written to disk :)

Reply:
Ah! I see. It hasn't mattered up to this point because there's only the only instance in the font, but yes, it makes sense to dedent it so it doesn't repeat repeatedly.